addons/mail/static/src/components/welcome_view/welcome_view.scss

```diff
@@ -0,0 +1,18 @@
+$button-size: 4rem;
```

Comment:
same remark as other, and move it to an actual variable file if you want it global, and make a more specific name